onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -3361,7 +3361,8 @@ void RegisterContribSchemas() {
           OpSchema::NonDifferentiable)
       .TypeConstraint(
           "T",
-          {"tensor(int8)",
+          {"tensor(bool)",
```

Comment:
I think this addition may need to be reflected in the contrib op docs: https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md

Reply:
i remember the opposite. had to generate and update the doc file. looks like the pipeline failed to confirm this: https://github.com/microsoft/onnxruntime/pull/24065/checks?check_run_id=38911854176